source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,210 @@
+Assuming we have set test retries to `2` (for a total of 3 attempts), the following is how the tests would run.
+
+1. A test runs for the first time. If the {% fa fa-check-circle green %} test passes, Cypress will move forward with any remaining tests as usual.
+
+2. If the {% fa fa-times red %} test fails, Cypress will tell you that the first attempt failed and will to run the test a second time.
+
+{% img /img/guides/test-retries/attempt-2-start.png %}
+
+3. If the {% fa fa-check-circle green %} test passes after the second attempt, Cypress will continue with any remaining tests.
+
+4. If the {% fa fa-times red %} test fails a second time, it will make the final third attempt to re-run the test.
+
+{% img /img/guides/test-retries/attempt-3-start.png %}
+
+5. If the {% fa fa-times red %} test fails a third time, Cypress will mark the test as failed and then move on to run any remaining tests.
+
+{% img /img/guides/test-retries/attempt-3-fail.png %}
+
+The following is a screen capture of what test retries looks like on the same failed test when run via {% url "`cypress run`" command-line#cypress-run %}.
+
+{% img /img/guides/test-retries/cli-error-message.png %}
+
+During {% url "`cypress open`" command-line#cypress-open %} you will be able to see the number of attempts made in the {% url "Command Log" test-runner#Command-Log %} and expand each attempt for review and debugging if desired.
+
+{% video local /img/guides/test-retries/attempt-expand-collapse-time-travel.mp4 %}
+
+# Configure Test Retries
+
+## Global Configuration
+
+If you want to change the number of attempts being made for all tests run for both `cypress run` and `cypress open`, you can configure this in your {% url "configuration file" command-line#cypress-open-config-file-lt-config-file-gt %} (`cypress.json` by default) by defining the `retries` property and giving it the desired number of retries.
+
+```jsx
+{
+  "retries": 1
+}
+```
+
+### Disable Test Retries
+
+If you would like to disable test retries completely (the default configuration), you can use the global configuration to do so by setting `retries` to 0.
+
+```jsx
+{
+  "retries": 0
+}
+```
+
+### Configure specific retries for different modes
+
+If you want to define different retry attempts for `cypress run` versus `cypress open`, you can pass the `retries` option an object with the following options:
+
+- `runMode` allows you to define the number of test retries when running `cypress run`
+- `openMode` allows you to define the number of test retries when running `cypress open`
+
+```jsx
+{
+  "retries": {
+    // Configure retries for `cypress run`
+    // Default is 0
+    "runMode": 2,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 1
+  }
+}
+```
+
+## Custom Configurations
+
+### Individual Test(s)
+
+If you want a specific test to run a custom number of retries, you can set this by using the {% url "test's configuration" writing-and-organizing-tests#Test-Configuration %}.
```

Comment:
"run a custom number of retries" 

what's the agreed upon way to say this? So far in this document I've seen:
- "number of times to retry a test if [it] fails"
- "number of test retries"
- "retry attempts"
- "number of attempts being made for [a test]"
- "suite of tests to re-run a custom number of times"

I think we would be causing unnecessary confusion by saying it all these different ways

Reply:
😅

Reply:
Great point @Bkucera! Going to try and restructure this.